dags/bqetl_analytics_aggregations.py

```diff
@@ -333,11 +353,19 @@
     )
 
     focus_ios_active_users_aggregates.set_upstream(wait_for_clients_last_seen_joined)
+    focus_ios_active_users_aggregates.set_upstream(wait_for_firefox_android_clients)
+    focus_ios_active_users_aggregates.set_upstream(
+        wait_for_firefox_ios_derived__firefox_ios_clients__v1
+    )
     focus_ios_active_users_aggregates.set_upstream(
         wait_for_search_derived__mobile_search_clients_daily__v1
     )
 
     klar_ios_active_users_aggregates.set_upstream(wait_for_clients_last_seen_joined)
+    klar_ios_active_users_aggregates.set_upstream(wait_for_firefox_android_clients)
+    klar_ios_active_users_aggregates.set_upstream(
+        wait_for_firefox_ios_derived__firefox_ios_clients__v1
```

Comment:
Also firefox_ios as upstream for klar, do you know why?

Reply:
Ok, it's because the upstream is bqetl_analytics_table.

Reply:
fwiw I also found this part confusing when reading through the code.